client/webpack/web_loaders/globals-loader.js

```diff
@@ -1,53 +1,79 @@
-var makeGlobals, stringify;
-
 var glob = require('glob')
 var path = require('path')
+var loaderUtils = require('loader-utils')
 
-var CLIENT_PATH = path.join(__dirname, '../../')
-var configFile = require('../../.config.json')
-var schemaFile = require('../../.schema.json')
-
-var manifests = glob.sync('*/bant.json', {
-  cwd: CLIENT_PATH,
-  realpath: true
-}).map(require)
+module.exports = function(globals) {
+  this.cacheable && this.cacheable()
 
-var modules = manifests.map(function(manifest) {
+  var clientPath = loaderUtils.parseQuery(this.query).clientPath
+  throwUnless(
+    clientPath,
+    '`clientPath` option is not set on query. Check your `globals-loader`' +
+    ' configuration in webpack.config.js'
+  )
 
-  var name = manifest.name === 'ide' ?
-    manifest.name.toUpperCase() :
-    manifest.name.charAt(0).toUpperCase() + manifest.name.slice(1)
+  var configFile = require(path.join(clientPath, '.config.json'))
+  throwUnless(
+    configFile,
+    'The generated config file does not exist. Did you run `./configure`' +
+    ' in koding root folder?'
+  )
 
-  name = name.replace(/-/g, '')
+  var schemaFile = require(path.join(clientPath, '.schema.json'))
+  throwUnless(
+    configFile,
+    'There is no bongo schema file on client folder. Did you run' +
```

Comment:
I think providing instructions for fixing dependency is a better message instead of asking whether user run a command.